spec/support/shared_examples/a_declarable.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.shared_examples 'a declarable' do
```

Comment:
The idea is that we can easily backfill the behaviour for declarable concerns in future and have both heading and commodity implementations have conforming implementations